src/util/utils.js

```diff
@@ -628,3 +630,31 @@ export const getYesterdayDateRoundDownHour = () => {
   const yesterday = date.setDate(date.getDate() - 1)
   return new Date(yesterday).toISOString()
 }
+
+export const getMonthName = (month: number, short: boolean = false) => {
+  return short ? MONTH_NAMES_SHORT[month] : MONTH_NAMES[month]
+}
+
+export const formatExpDate = (expiration: Date, short: boolean = false) => {
+  const expirationDate = new Date(expiration)
+  return `${getMonthName(expirationDate.getMonth())}  ${expirationDate.getDate()}, ${expirationDate.getFullYear()}`
+}
+
+export const networkAlert = (): boolean => {
+  return Alert.alert(s.strings.network_alert_title, s.strings.network_alert_text)
+}
+
+export const getFeeDisplayed = (number: number) => {
+  const defaultAmount = 2
+  const dec = number % 10
+
+  if (dec) {
+    return dec.toString().length > 2 ? number.toString() : number.toFixed(defaultAmount)
+  }
+
+  return number.toFixed(defaultAmount)
+}
+
+export const validateFioAddress = (fioAddress: string): boolean => {
```

Comment:
This should be provided by the currency plugin and exported via otherMethods

Reply:
ok